docs/httplug.md

```diff
@@ -5,22 +5,35 @@ Httplug is an abstraction for HTTP clients. There are two main use cases:
 1. Usage in a project
 2. Usage in a reusable package
 
-In both cases, the client provides a `sendRequest` method to send a PSR-7 `RequestInterface` and returns a PSR-7 `ResponseInterface` or throws an exception that implements `Http\Client\Exception`.
+In both cases, the client provides a `sendRequest` method to send a PSR-7 `RequestInterface` and returns a PSR-7 `ResponseInterface`
+or throws an exception that implements `Http\Client\Exception`.
 
-See the [tutorial](tutorial.md) for a concrete example.
+There is also the HttpAsyncClient, available in [php-http/httplug-async](https://packagist.org/packages/php-http/httplug-async), which provides the `sendAsyncRequest` method to send a request asynchronously and returns a `Http\Client\Promise`.
+It can be used later to retrieve a PSR-7 `ResponseInterface` or an exception that implements `Http\Client\Exception`.
+
+Contract for the HttpAsyncClient is still experimental and will be merged into Httplug repository once we considered it stable.
```

Comment:
> You mean move the client to the main contract and use the PSR promise, right?

I think he was talking when we will move async client to the main contract, yes. With the current namespace, if we use a new interface it will assure a BC break as we will use the same namespace.

But i don't think Promise PSR will be here soon, so we can always throws a 2.0 version at this time, WDYT ?